crates/dbsp/src/circuit/circuit_builder.rs

```diff
@@ -4170,6 +4222,12 @@ where
     fn fixedpoint(&self, scope: Scope) -> bool {
         unsafe { (*self.operator.get()).fixedpoint(scope) }
     }
+
+    fn commit(&self, _cid: usize) -> Result<(), Error> {
+        //unsafe { (*self.operator.get()).commit(cid) }
+        // why I get two calls for this crap?
```

Comment:
?

Reply:
If I uncomment this call, every spine instance gets two calls to commit() for every commit command that is sent to the circuit. I was hoping that maybe @ryzhyk knows why and how to avoid this.

Reply:
(when debugging it looked like the same spine instance appears in FeedbackInputNode as well as FeedbackOutputNode?)

Reply:
Oh, OK, I'll leave this to him. (And otherwise, it would be good to make the comment more informative.)

Reply:
Oh, I see, you did.